region/lowernorfair/east.json

```diff
@@ -1535,6 +1535,12 @@
           "note": "Technically you can get the drops but you likely won't"
         }
       ],
+      "reusableRoomwideNotable": [
+        {
+          "name": "Amphitheatre Reverse Acid Dive",
+          "note": "Dive into the acid and wall jump up the left wall to the doorway."
```

Comment:
There will be reverse acid dive strats that do not use walljump, like spacejump.